### ISSUE TITLE
Channel/World crash on heavy load.

### ISSUE THREAD

**Issue by alkalime** [comphack/comp_hack#327]
world.log
```
[2017/10/01 19:54:30] ERROR: Invalid session key provided for account with username 'username'
[2017/10/01 19:54:30] DEBUG: Client disconnect: 127.0.0.1
[2017/10/01 19:54:30] WARNING: 53 user(s) forcefully logged out from channel 0.
[2017/10/01 19:58:04] 
Bye!
```

channel.log
```
[2017/10/01 19:54:30] ERROR: Option selected for unknown event: <username>
[2017/10/01 19:54:30] 0000 {b3}29 00 00 7c b6 07 43  2a c6 85 c3 7c b6 87 42  .)..|..C *...|..B
0010  2a c6 05 c3 ff ff 95 43  1d da 2f 42 1d da 31 42  *......C ../B..1B
[2017/10/01 19:54:30] ERROR: Invalid response received for AccountLogin.
[2017/10/01 19:54:30] =========================================================================
[2017/10/01 19:54:30] 0000 {00}00 00 00                                       .... 
[2017/10/01 19:54:30] 0000 {00}00 00 00                                       .... 
[2017/10/01 19:54:30] =========================================================================
[2017/10/01 19:54:30] ERROR: Failed to process message!
[2017/10/01 19:54:30] World connection closed. Shutting down.
[2017/10/01 19:54:30] =========================================================================
[2017/10/01 19:54:30] =========================================================================
[2017/10/01 19:54:30] 0000 {c7}2f 57 54                                       ./WT 
[2017/10/01 19:54:30] =========================================================================
[2017/10/01 19:54:30] 
Bye!
```

**Comment by alkalime** [comment]
Just experienced this bug again (with 44 users). It caused the lobby to go down too this time, but there was nothing in the lobby.log file that would indicate why.

**Comment by alkalime** [comment]
Managed to talk to a user whose account caused this to happen. They got logged in on a new character (act 0), got past character select and were able to walk around for a few seconds before the server crashed.

**Comment by alkalime** [comment]
Fixed by #358
